coral/src/app/features/connectors/browse/components/ConnectorTable.tsx

```diff
@@ -28,18 +30,18 @@ function ConnectorTable(props: ConnectorTableProps) {
     {
       type: "custom",
       headerName: "Connector",
-      UNSAFE_render: ({ connectorName }: ConnectorTableRow) => (
-        <a href={`/connectorOverview?connectorName=${connectorName}`}>
+      UNSAFE_render: ({ connectorName, environmentId }: ConnectorTableRow) => (
```

Comment:
We should have that behind a feature flag, same as with the links for topic overview.